Clean up on the List and List-Method Isles (#2404)
exercism/python: exercises/concept/chaitanas-colossal-coaster/.docs/introduction.md

[-2, 0, 1, 2, 3]
```

> `list.extend()` can be used to combine an existing list with the elements from another iterable (for example, a `set`, `tuple`, `str`, or `list`).  The iterable is _unpacked_ and elements are appended in order. (_using `.append()` in this circumstance would add the entire iterable as a **single item**._).

Reviewer: ```suggestion
`list.extend()` can be used to combine an existing list with the elements from another iterable (for example, a `set`, `tuple`, `str`, or `list`).  The iterable is _unpacked_ and elements are appended in order. (_Using `.append()` in this circumstance would add the entire iterable as a **single item**._).
```

> `list.extend()` can be used to combine an existing list with the elements from another iterable (for example, a `set`, `tuple`, `str`, or `list`).  The iterable is _unpacked_ and elements are appended in order. (_using `.append()` in this circumstance would add the entire iterable as a **single item**._).

Author: ```suggestion
`<list>.extend()` can be used to combine an existing list with the elements from another iterable (for example, a `set`, `tuple`, `str`, or `list`).  The iterable is _unpacked_ and elements are appended in order. (_Using `<list>.append()` in this circumstance would add the entire iterable as a **single item**._).
```

## Reversing and reordering

> The `list.reverse()` method will reverse the order of elements.

Reviewer: ```suggestion
The `list.reverse()` method will reverse the order of elements in-place.
```

Author: Not sure why I can't auto-commit this.  Going to self-suggest.  Grr.

> The `list.reverse()` method will reverse the order of elements.

Author: ```suggestion
The `<list>.reverse()` method will reverse the order of elements in-place.
```

> The `list.reverse()` method will reverse the order of elements.

Author: ```suggestion
The `<list>.reverse()` method will reverse the order of elements in-place.
```

[3, 2, 1]
```

> A list can be re-ordered _**in place**_ with the help of `list.sort()`. Internally, Python uses [`Timsort`][timsort] to arrange the list. The default order will be _ascending_. Take a look at the Python docs for some [additional tips and techniques for sorting][sorting how to] lists effectively.

Author: ```suggestion
A list can be re-ordered _**in place**_ with the help of `<list>.sort()`. Internally, Python uses [`Timsort`][timsort] to arrange the list. The default order will be _ascending_. Take a look at the Python docs for some [additional tips and techniques for sorting][sorting how to] lists effectively.
```

## Finding the index of items

> `list.index()` will return the index number of the _first occurrence_ of an item passed in. If there are no occurrences, a `ValueError` is raised. If the exact position of an item isn't needed, the built-in `in` operator is more efficient for checking simple membership.

Reviewer: ```suggestion
`list.index()` will return the index number of the _first occurrence_ of an item passed in. If there are no occurrences, a `ValueError` is raised. If the exact position of an item isn't needed, the built-in `in` operator is more efficient for checking if a list contains that value.
```

Author: Noticed a couple other edits needed, so did those in a self-edit that included these.  Going to "resolve" this.

> `list.index()` will return the index number of the _first occurrence_ of an item passed in. If there are no occurrences, a `ValueError` is raised. If the exact position of an item isn't needed, the built-in `in` operator is more efficient for checking simple membership.

Author: ```suggestion
`<list>.index(<item>)` will return the index number of the _first occurrence_ of an item passed in. If there are no occurrences, a `ValueError` is raised. If the exact position of an item isn't needed, the built-in `in` operator is more efficient for checking if a list contains a given value.
```
